Mathlib/Topology/VectorBundle/Basic.lean

```diff
@@ -60,7 +60,7 @@ Vector bundle
 
 noncomputable section
 
-open Bundle Set Classical
+open Bundle Set Classical Topology
```

Comment:
This line isn't in on the port-status page.

Reply:
If it's needed
```suggestion
open scoped Bundle Set Classical Topology
```

Reply:
I need it for the `Continuous[_, _]` notation. IMHO, it's more readable than `@Continuous _ _ _ _ f`

```suggestion
open Bundle Set Classical
open scoped Topology
```